src/commands/login.rs

```diff
@@ -1,29 +1,121 @@
+use anyhow::format_err;
+use opener;
+use serde::Deserialize;
+use std::path::PathBuf;
+use std::{thread, time};
 use structopt::StructOpt;
```

Comment:
```suggestion
use std::path::PathBuf;
use std::thread;
use std::time;

use anyhow::format_err;
use opener;
use serde::Deserialize;
use structopt::StructOpt;
```

We conventionally sort std imports in a different group on their own so that they stand out!